packages/google/src/google-provider.ts

```diff
@@ -103,8 +103,9 @@ export function createGoogleGenerativeAI(
       generateId: options.generateId ?? generateId,
       supportedUrls: () => ({
         '*': [
-          // HTTP URLs:
-          /^https?:\/\/.*$/,
+          // Only allow requests to the Google Generative Language "files" endpoint
+          // e.g. https://generativelanguage.googleapis.com/v1beta/files/...
+          new RegExp(`^${baseURL}/files/.*$`),
```

Comment:
this would also apply to images - are web url images not supported?

Reply:
yes web url images are not supported, only image links with that prefix are supported unfortunately.